src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/StyledCard/styles.js

```diff
@@ -21,9 +25,11 @@ export const useStyles = makeStyles(theme => ({
   cardSubheader: {
     color: theme.palette.text.darkShadedGray,
     fontWeight: theme.typography.fontweight,
-    fontSize: 20,
-    letterSpacing: 0.25,
+    fontSize: 18,
```

Comment:
We could extract the font sizes into constants and have consistent typography. We could also consider building components for `Heading`, `Subheading`, `Description` and `Text` which have pre defined size and spacing. Will be much easier to maintain and will give a more consistent look and feel to the app.

Reply:
will speak with design team reg this...

Reply:
connected to Greg reg this and created an [issue](https://github.com/singnet/refactored-dapp/issues/37)